src/DigitalProductPassport/FoodArtifact/DataSheet_v0.1.py

```diff
@@ -0,0 +1,95 @@
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
```

Comment:
@hsuomi: Which name should this one use; `DigitalProductPassport/FoodArtifact/Nutritrional` [sic] or `DigitalProductPassport/FoodArtifact/DataSheet`? The first one is the name filled in on the sheet, the latter is the name of the sheet with the content. And also note the typo in the first one that I mentioned in a comment in the sheet already yesterday.